src/twisted/web/test/test_flatten.py

```diff
@@ -237,13 +258,13 @@ def test_serializedAttributeWithTagWithAttribute(self):
             XML(flattened).attrib["src"], b'<a href="&lt;&gt;&amp;&quot;"></a>'
         )
 
-    def test_serializeComment(self):
+    def test_serializeComment(self) -> None:
         """
         Test that comments are correctly flattened and escaped.
         """
         self.assertFlattensImmediately(Comment("foo bar"), b"<!--foo bar-->")
 
-    def test_commentEscaping(self):
+    def test_commentEscaping(self) -> Deferred[Any]:
```

Comment:
`Deferred[object]` perhaps.

Reply:
`Deferred[object]` is actually not accepted by mypy. But in the error message, it tells me the actual type is `Deferred[List[bytes]]`, so I'll use that.

Reply:
I'm a little surprised about that, but ok